examples/02_Parameter Estimation.ipynb

```diff
@@ -473,17 +468,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Notice that by using data from multiple runs, we are able to produce a lower AME Error than before! This is because we are able to simulate the noise multiple times, which in turn, allows our `estimate_params()` to produce a more accurate result since it is given more data to work with!"
+    "Notice that by using data from multiple runs, we are able to produce a lower AME Error than before. This is because we are able to simulate the noise multiple times, which in turn, allows our `estimate_params()` to produce a more accurate result since it is given more data to work with."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simplified Battery"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Example 4) Simplified Battery\n",
     "The previous examples all used a simple model, the ThrownObject. For large, complex models, it can be VERY difficult and computationall expensive.\n",
```

Comment:
```suggestion
    "The previous examples all used a simple model, the ThrownObject. For large, complex models, it can be VERY difficult and computationally expensive.\n",
```